.github/workflows/go.yml

```diff
@@ -0,0 +1,38 @@
+name: Test
+
+on:
+  push:
+    branches: 
+      - '*'
+  pull_request:
+    branches: 
+      - '*'
+
+jobs:
+  build:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get dependencies
+      run: go get -v -t -d ./...
```

Comment:
This is the pre go-modules way of doing this. 
Should not be necessary at all (go test does this I believe) otherwise I think you want `go mod download`